source/manual/alerts/data-sync-failed-to-munge.html.md

```diff
@@ -0,0 +1,29 @@
+---
+owner_slack: "#govuk-2ndline"
+title: URLs in data sync failed to munge
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-06-25
+review_in: 3 months
+---
+
+As part of the data syncs, the router data should update to point to the relevant environment.
+
+If this fails, then on integration for example, apps will appear to be broken - you'll likely see `cache-1.router.integration.publishing.service.gov.uk;5xx rate for www-origin` alerts and 5xx's for frontend apps.
+
+Check URLs in router-api:
+
+```
+ssh router-backend-1.staging
+
+govuk_app_console router-api
+
+pp Backend.pluck(:backend_url)
+```
+
+If URLs don't have `staging` then the `scripts/router-munge-hostnames.sh` in `env-sync-and-backup` repo failed for some reason.
```

Comment:
Probably want to tweak this to be staging or integration. You also will want to phrase this as "most of the URLs" as some of our backends (for third party things) we don't do this.

Reply:
👍